api/bin/chainflip-lp-api/src/main.rs

```diff
@@ -63,13 +63,22 @@ pub trait Rpc {
 	#[method(name = "register_account")]
 	async fn register_account(&self) -> RpcResult<Hash>;
 
+	#[deprecated(note = "Use `request_liquidity_deposit_address_v2` instead")]
 	#[method(name = "liquidity_deposit")]
 	async fn request_liquidity_deposit_address(
 		&self,
 		asset: Asset,
 		wait_for: Option<WaitFor>,
 		boost_fee: Option<BasisPoints>,
-	) -> RpcResult<ApiWaitForResult<String>>;
+	) -> RpcResult<ApiWaitForResult<AddressString>>;
+
+	#[method(name = "liquidity_deposit_v2")]
+	async fn request_liquidity_deposit_address_v2(
```

Comment:
nit: It's not clear why we have a different rpc name than the method name.

We could take advantage of this and just use the more explicit naming for the new method (which is a little more consistent with other methods, ie. `{verb}_{thing}`):

 ```suggestion
	#[method(name = "request_liquidity_deposit_address")]
	async fn request_liquidity_deposit_address(
```

(And rename the old method as `request_liquidity_deposit_address_legacy` or something).